package.json

```diff
@@ -45,8 +45,8 @@
   "homepage": "https://github.com/airbnb/hypernova-react#readme",
   "peerDependencies": {
     "hypernova": "^2.0.0",
-    "react": "0.14.x || >= 15.x",
-    "react-dom": "0.14.x || >= 15.x"
+    "react": "~0.14 || ^15 || ^16",
+    "react-dom": "~0.14 || ^15 || ^16"
```

Comment:
`^` for versions below 1.0 only updates the patch, so `^0.14` is correct here.

Reply:
updated, thanks for bearing with me on this. Really, I'm gaining knowledge here!